### DISCUSSION TITLE
How to get the width and height of upload

### DISCUSSION THREAD

**Discussion by blueMountain007** [payloadcms/payload#1036] [Q&A]
How to get the width and height of upload

**Comment by blueMountain007** [comment]
Now in order to get the width and height of the upload, I have to do it awkwardly via imageSizes：

<img width="851" alt="image" src="https://user-images.githubusercontent.com/16400882/187677040-ba6b8bb9-5faa-4424-ac16-8ad134d76288.png">

**Comment by blueMountain007** [comment]
After I set the imageSize field to null, I found that the image corresponding to imageSize does not exist in s3： 
<img width="776" alt="image" src="https://user-images.githubusercontent.com/16400882/188041094-50483927-3380-4aaf-98e8-d125f9c8057f.png">

**Reply by jmikrut** [reply]
Hey @zhounewz — I believe this is a duplicative issue but we will try to reproduce this comment in specific. Give @denolfe and I a bit here to take a look!
